### PR TITLE
IBX-5054: HTML code gets removed from the Custom tag's string attribute

### PR DESCRIPTION
**Pull request by vidarl**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [IBX-5054](https://issues.ibexa.co/browse/IBX-5054)
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | 4.3
| **BC breaks**      | no
| **Tests pass**     | yes/no
| **Doc needed**     | no

Missing escaping of input

**TODO**:
- [x] Implement feature / fix a bug.
- [ ] ~Implement tests.~ ( Assume this is not needed for this one ?)
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

**Comment by vidarl** [comment]
Failing SOLR test is not related to changes in this PR...

File: src/bundle/Resources/public/js/CKEditor/custom-tags/block-custom-tag/custom-tag-editing.js [hunk 1]
```diff
@@ -76,7 +76,12 @@ class IbexaCustomTagEditing extends Plugin {
                     const domElement = this.toDomElement(domDocument);
 
                     domElement.innerHTML = Object.entries(modelElement.getAttribute('values')).reduce((total, [attribute, value]) => {
-                        const attributeValue = value !== null ? value : '';
+                        // Escaping
+                        // <script>alert("Hello! I am a script!");</script> --> &lt;script&gt;alert("Hello! I am a script!");&lt;/script&gt;
+                        const stringTempNode = domDocument.createElement('div');
```
Comment (lucasOsti) [inline src/bundle/Resources/public/js/CKEditor/custom-tags/block-custom-tag/custom-tag-editing.js:81]: What do you think about escaping without creating div?
```
const valueString = value ?? '';
const attributeValue = valueString
	.replace(/&/g, "&amp;")
        .replace(/</g, "&lt;")
        .replace(/>/g, "&gt;")
        .replace(/"/g, "&quot;")
        .replace(/'/g, "&#39;");
```

**Review by dew326 — APPROVED** [review]

**Review by Gengar-i — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

File: src/bundle/Resources/public/js/CKEditor/custom-tags/block-custom-tag/custom-tag-editing.js (hunk 1 above)
Comment (vidarl) [reply src/bundle/Resources/public/js/CKEditor/custom-tags/block-custom-tag/custom-tag-editing.js:81]: Escaping like I currently do seems to be quite common in JS. But if you prefer, I can change it as suggested. However, others have already approved it so not sure what others think of it.

I am not sure either if your suggestion covers all characters that needs to be escaped ?

**Review by lucasOsti — APPROVED** [review]

File: src/bundle/Resources/public/js/CKEditor/custom-tags/block-custom-tag/custom-tag-editing.js (hunk 1 above)
Comment (lucasOsti) [reply src/bundle/Resources/public/js/CKEditor/custom-tags/block-custom-tag/custom-tag-editing.js:81]: After talking with the frontend team, I approved for your solution :)

**Review by tischsoic — APPROVED** [review]

**Review by micszo — APPROVED** [review]
Retest of original issue OK.

Sanities on rich-text field OK.

QA Approved on Ibexa Experience 4.3.4.

**Comment by alongosz** [comment]
Rebased to see if we can get CI green.

**Pull request merged**